Assets/GameAssets/Player/PlayerScripts/PlayerHealth.cs

```diff
@@ -83,4 +91,14 @@ public override void OnDeath()
         base.OnDeath();
         PlayerManager.Instance.InvokeOnPlayerDeath();
     }
+
+    private void SubscribeToEvents()
```

Comment:
For some functions I get not having XML as they are self explanatory such as Awake, although I've had people ask for XML comments on my subscribe to events functions, so now I'm asking you to do that. Though since Andrea didn't comment on this I dunno. I suppose do it or not 🤷‍♂️